72991: Added screenshot. Fixed passive language. Moved note. All for asigra. (#668)
freenas/freenas-docs: userguide/services.rst

For the initial backend configuration, click
:menuselection:`Services --> Asigra DS-System`. When prompted to choose
the :guilabel:`Base Filesystem`, select the dataset to store the Asigra

> backups then click :guilabel:`OK`. The system automatically creates

Reviewer: Take a breath here, and "the system" is ambiguous.  Fortunately, it is not needed:
```suggestion
backups, then click :guilabel:`OK`. Any required database entries are created and the service is started.
```

> the required database entries and starts the service.

Reviewer: ```suggestion

```

   the application to open *DSOP.jnlp* with, select Java Web Start
   Launcher (:command:`javaws`).

While the service is running, the :guilabel:`Open DS-Operator Web Interface`

> button appears in :menuselection:`Services --> Asigra DS-System`. Click

Reviewer 2: ```suggestion
button appears in
:menuselection:`Services --> Asigra DS-System`.
Click :guilabel:`Open DS-Operator Web Interface` to download and launch the
```

Reviewer 2: We're supposed to put :menuselection:`` items on their own line now (confirm with Warren)

> :guilabel:`Open DS-Operator Web Interface` to download and launch the

Reviewer 2: ```suggestion

```

   the application to open *DSOP.jnlp* with, select Java Web Start
   Launcher (:command:`javaws`).

While the service is running, the :guilabel:`Open DS-Operator Web Interface`
button appears in
:menuselection:`Services --> Asigra DS-System`.
Click :guilabel:`Open DS-Operator Web Interface` to download and launch the
Asigra management application.

.. _asigra settings:

.. figure:: images/services-asigra.png
   :width: 50%

   Asigra settings

> Contact Asigra for further documentation on using DS-Operator.

Reviewer: Can we add a link here?
